paper/paper.md

```diff
@@ -0,0 +1,78 @@
+---
+title: 'OpenTURNS: A Python package for uncertainty quantification'
+tags:
+  - Python
+  - uncertainty quantification
+  - probabilistic modeling
+  - statistics
+  - meta-modeling
+  - calibration
+  - stochastic processes
+  - Bayesian modeling
+  - machine learning
+  - supervised machine learning
+  - unsupervised machine learning
+  - surrogate
+authors:
+  - name: Michaël Baudin
+    orcid: 0000-0002-0450-7947
+    affiliation: 1
+  - name: Anne Dutfoy
+    affiliation: 1
+  - name: Joseph Muré
+    affiliation: 1
+  - name: Julien Pelamatti
+    affiliation: 1
+  - name: Régis Lebrun
+    affiliation: 2
+  - name: Sofiane Haddad
+    affiliation: 2
+  - name: Julien Schueller
+    affiliation: 3
+  - name: Loïc Brevault
+    affiliation: 4
+  - name: Mathieu Balesdent
+    affiliation: 4
+affiliations:
+ - name: EDF R&D, France
+   index: 1
+ - name: Airbus, France
+   index: 2
+ - name: Phimeca, France
+   index: 3
+ - name: ONERA, France
+   index: 4
+date: 30 April 2025
+bibliography: paper.bib
+---
+
+# Summary
+
+``OpenTURNS`` is a C++ library for uncertainty propagation by probabilistic methods.
+OpenTURNS is also available as a Python module. 
+
+## Introduction
+Numerical simulation and the uncertainties associated with these models make decision-making robust in the design and operation of complex systems. Supervisory authorities in safety-critical industrial sectors (e.g. nuclear power or civil aviation) are demanding that players use tried-and-tested methods, backed up by tested, documented and auditable implementations. As a result, several uncertainty processing libraries have emerged over the last few decades. However, many of these approaches are dedicated to a particular discipline, such as numerical fluid mechanics or hydrology. This approach limits the sharing of best practices. This is why a generic approach to uncertainty management is so important.
+
+## Overview
+OpenTURNS is an open-source software under the LGPL license based on a C++ library and provides a Python package. OpenTURNS is a generic library for probabilistic modeling and uncertainty quantification. The library provides full documentation, in particular of the programming interface, examples and probabilistic theory. In Python, plotting features are provided using Matplotlib. 
+
+## Features
```

Comment:
Should we add somewhere in the Features that it is possible to "wrap/interface with" any "simulation model/codes" that can be evaluated in Python ? 
It can be important to the interested user to understand that it is possible to carry out UQ with OpenTURNS on any code/model if a standard "wrapping" is carried out (through PythonFunction or otwrapy) ?

Reply:
Very true. I added a paragraph on this crucial topic.